appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java

```diff
@@ -5119,6 +5123,39 @@ String newerVersionComponentException(String componentType, int srcCompVersion,
   @Description("")
   String MaximumRangeMethods();
 
+  @DefaultMessage(
+    "<table border='1' cellpadding='8' cellspacing='0'>" +
+    "<thead>" +
+    "<tr>" +
+    "<th>Action</th>" +
+    "<th>Key Combination</th>" +
+    "</tr>" +
+    "</thead>" +
+    "<tbody>" +
+    "<tr><td>Focus Component search box</td><td>/</td></tr>" +
+    "<tr><td>Focus Components tree</td><td>T</td></tr>" +
+    "<tr><td>Focus Viewer</td><td>V</td></tr>" +
+    "<tr><td>Focus Properties Panel</td><td>P</td></tr>" +
+    "<tr><td>Switch between Designer and Block editor</td><td>Ctrl + Alt</td></tr>" +
+    "<tr><td>Rename Component</td><td>Alt + N</td></tr>" +
+    "<tr><td>Reset Connection</td><td>Alt + Shift + R</td></tr>" +
+    "<tr><td>Refresh Companion Screen</td><td>Alt + R</td></tr>" +
+    "<tr><td>Open/Close Backpack</td><td>Alt + B</td></tr>" +
+    "<tr><td>Switch between Inline and External Inputs</td><td>Alt + I</td></tr>" +
+    "<tr><td>Add comment to a block</td><td>Alt + K</td></tr>" +
+    "<tr><td>Expand/Collapse block</td><td>Alt + O</td></tr>" +
+    "<tr><td>Navigate up the blocks</td><td>Alt + S</td></tr>" +
+    "<tr><td>Navigate down the blocks</td><td>Alt + W</td></tr>" +
+    "<tr><td>Zoom in</td><td>Alt + +</td></tr>" +
+    "<tr><td>Zoom out</td><td>Alt + -</td></tr>" +
+    "<tr><td>Re-center</td><td>Alt + G</td></tr>" +
+    "<tr><td>Navigate Components</td><td>↑/↓</td></tr>" +
+    "<tr><td>Open this dialog</td><td>Alt + ?</td></tr>" +
+    "</tbody>" +
+    "</table>")
```

Comment:
As per our conversation, the Delete/Backspace shortcut to delete a component should be included here, even though it is not part of this project.

Multiselect and Copy-Paste in the designer should also be included. I can do that myself.